.github/workflows/prerelease.yml

```diff
@@ -37,17 +37,22 @@ jobs:
         cd "${GITHUB_WORKSPACE}/ReleaseTooling"
         swift run manifest --sdk-repo-url "${GITHUB_WORKSPACE}" --output-file-path ./output.json --for-gha-matrix-generation
         echo "::set-output name=matrix::{\"include\":$( cat output.json )}"
-    - name: Get token
-      run: |
-         scripts/decrypt_gha_secret.sh scripts/gha-encrypted/oss-bot-access.txt.gpg \
-         oss-bot-access.txt "$bot_token_secret"
     - name: Update SpecsTesting repo setup
       run: |
-         ossbotaccess=`cat oss-bot-access.txt`
-         BOT_TOKEN="${ossbotaccess}" test_version="${nightly_version}" sdk_version_config="${GITHUB_WORKSPACE}/scripts/create_spec_repo/RC_firebase_sdk.textproto" local_sdk_repo_dir="${local_sdk_repo_dir}" podspec_repo_branch="${podspec_repo_branch}" scripts/release_testing_setup.sh prerelease_testing
+         test_version="${nightly_version}" \
+           sdk_version_config="${GITHUB_WORKSPACE}/scripts/create_spec_repo/RC_firebase_sdk.textproto" \
+           local_sdk_repo_dir="${local_sdk_repo_dir}" \
+           podspec_repo_branch="${podspec_repo_branch}" \
+           scripts/release_testing_setup.sh prerelease_testing
+      env:
+         BOT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Get token
+      run: |
+         scripts/decrypt_gha_secret.sh scripts/gha-encrypted/prerelease-testing-token.txt.gpg \
+         prerelease-testing-token.txt "$bot_token_secret"
     - name: Clean spec repo
       run: |
-         ossbotaccess=`cat oss-bot-access.txt`
+         ossbotaccess=`cat prerelease-testing-token.txt`
```

Comment:
Can this leak?

Reply:
No, it doesn't. See here: https://github.com/firebase/firebase-ios-sdk/actions/runs/9768323572/job/26965562893#step:6:2

Reply:
That being said, the more I work on this, I don't like the pattern having a personal access token (PAT) encrypted in gpg format. They're harder to work with and more vulnerable because GitHub won't stop you from adding an `echo $ossbotaccess` and leaking it.